scripts/Jenkinsfile

```diff
@@ -10,111 +10,111 @@ node {
   def testContainer = "${job_name}${env.BUILD_NUMBER}_test"
   def imageName = "acceptance-tests-${job_name}${env.BUILD_NUMBER}"
   def envFile = ".env"
+  def qaseEnvFile = ".qase.env"
   def branch = "main"
-  if ("${env.BRANCH}" != "null" && "${env.BRANCH}" != "") {
-    branch = "${env.BRANCH}"
+  if (env.BRANCH && env.BRANCH != "null") {
+    branch = env.BRANCH
   }
 
   def repo = scm.userRemoteConfigs
-  if ("${env.REPO}" != "null" && "${env.REPO}" != "") {
-    repo = [
-      [url: "${env.REPO}"]
-    ]
+  if (env.REPO && env.REPO != "null") {
+    repo = [[url: env.REPO]]
   }
   def timeout = "60m"
-  if ("${env.TIMEOUT}" != "null" && "${env.TIMEOUT}" != "") {
-    timeout = "${env.TIMEOUT}"
+  if (env.TIMEOUT && env.TIMEOUT != "null") {
+    timeout = env.TIMEOUT
   }
+
   wrap([$class: 'AnsiColorBuildWrapper', 'colorMapName': 'XTerm', 'defaultFg': 2, 'defaultBg': 1]) {
     withFolderProperties {
-      paramsMap = []
-      params.each {
-        if (it.value && it.value.trim() != "") {
-          paramsMap << "$it.key=$it.value"
-        }
-      }
+      def paramsMap = params.findAll { it.value?.trim() }.collectEntries { [(it.key): it.value] }
       withCredentials([
-
         string(credentialsId: 'AWS_ACCESS_KEY_ID', variable: 'AWS_ACCESS_KEY_ID'),
         string(credentialsId: 'AWS_SECRET_ACCESS_KEY', variable: 'AWS_SECRET_ACCESS_KEY'),
         string(credentialsId: 'AWS_SSH_PEM_KEY', variable: 'AWS_SSH_PEM_KEY'),
         string(credentialsId: 'ADMIN_PASSWORD', variable: 'ADMIN_PASSWORD'),
-        string(credentialsId: 'RKE2_RHEL_PASSWORD', variable: 'RKE2_RHEL_PASSWORD')
+        string(credentialsId: 'RKE2_RHEL_PASSWORD', variable: 'RKE2_RHEL_PASSWORD'),
+        string(credentialsId: 'QASE_AUTOMATION_TOKEN', variable: 'QASE_AUTOMATION_TOKEN')
```

Comment:
Just to clarify, this `QASE_AUTOMATION_TOKEN` is coming from the user?

Reply:
Can we add a doc for this? I can't see any reference for it

Reply:
So this needs to be set when running locally only?

Reply:
yes, have added on pkg readme